src/cluster.c

```diff
@@ -783,24 +784,29 @@ unsigned long getClusterConnectionsCount(void) {
  * { and } is hashed. This may be useful in the future to force certain
  * keys to be in the same node (assuming no resharding is in progress). */
 unsigned int keyHashSlot(char *key, int keylen) {
-    int s, e; /* start-end indexes of { and } */
+    if (server.cluster_custom_hash_enable) {
+        return CustomkeyHashSlot(key,keylen);
+    }
+    else {
```

Comment:
I do not have the strong reason...
But if we do the return in the `if` block, is the else block still needed here?
I think that we can reduce some changes and code indentation